src/utils/doctypes.js

```diff
@@ -0,0 +1,12 @@
+export const DOCTYPE_GEOJSON = 'io.cozy.timeseries.geojson'
+export const DOCTYPE_ACCOUNTS = 'io.cozy.accounts'
```

Comment:
Why moving those consts here, when all others constants are in the same consts.js file?

Reply:
pour coller aux autres app, les constantes relatives aux doctypes sont avec les autres sujets qui concernent les doctypes, dans un fichier spécifiques

Reply:
Soit, mais je trouve ça dommage d'avoir toutes les constantes de l'app à un même endroit SAUF celles des doctypes, parce qu'on fait comme ça dans d'autres apps :/
Si je suis un dev qui ne connaît pas les habitudes Cozy, je vais trouver ça incohérent